File: programme/volume-of-sphere/volume-of-sphere.c
// Write a programme to compute the volume of a Sphere
#include <stdio.h>
#include <math.h>

int main()
{
    double radius;
    printf("Enter the radius : ");
    scanf("%lf", &radius);
    double area = (4 / 3.0) * M_PI * radius * radius * radius;
    printf("\nThe volume of sphere : %.2lf\n", area);
    return 0;
}

File: programme/volume-of-sphere/VolumeOfSphere.cs
// Write a programme to compute the volume of a Sphere
using System;

namespace Geometry
{
    public class VolumeOfSphere
    {
        public static double CalculateVolume(double radius)
        {
            return Math.Pow(radius, 3) * Math.PI * (4.0 / 3.0);
        }

        public static void Main()
        {
            Console.Write("Radius : ");

            if (double.TryParse(Console.ReadLine(), out double radius))
            {
                double volume = CalculateVolume(radius);
                Console.WriteLine("Volume : " + Math.Round(volume, 2));
            }
            else
            {
                Console
                    .WriteLine("The value provided for the radius is invalid");
            }
        }
    }
}
